### DISCUSSION TITLE
Closed Windows List

### DISCUSSION THREAD

**Discussion by beefjerky9** [onemen/TabMixPlus#287] [Ideas]
I see there is an option for a Closed Windows List in the tools menu, but that is awkward to get to. Could this be added as an option to the Tab Context Menu and Main Context Menu? It would be super helpful if it could.

Thanks!

**Comment by onemen** [comment]
You have it in History menu

**Reply by beefjerky9** [reply]
Okay, still less convenient than in the tab context or main context menu. If it can't be done, that's fine. It's just a request.

**Comment by pyrates999** [comment]
Right click on any tab and you have a closed list menu.

**Reply by beefjerky9** [reply]
That's only the list of closed tabs, not the closed window list that I'm referring to. Unless your Firefox behaves differently, or maybe you have another add-on that's doing it?

**Reply by pyrates999** [reply]
Ah you're right, sorry.
